orangecontrib/geo/widgets/owgeocoding.py

```diff
@@ -26,12 +28,20 @@ def available_name(domain, template):
         if name not in domain:
             return name
 
+def guess_region_attr_name(data):
+    """Return the name of the first variable that could specify a region name"""
+    var = next(chain((var for var in data.domain.metas if var.is_string or
+                      var.is_discrete),
+                     (var for var in data.domain.variables if
+                      var.is_discrete)),
```

Comment:
It's customary to break long comprehension lines at (before) if.

For the record, @astaric's proposed version was better.